R/class-workbook.R

```diff
@@ -805,6 +806,7 @@ wbWorkbook <- R6::R6Class(
     add_data = function(
         sheet           = current_sheet(),
         x,
+        dims            = NULL,
         startCol        = 1,
```

Comment:
`dims` after `startCol`, `startRow` seems better when using unnamed arguments:

```r
wb$add_data(sheet, data, 1, 1) # col, row
wb$add_data(sheet, data, range = c(1, 1) # explicitly use range
```

Reply:
yeah, I was looking for a good spot, after startCol, startRow is a good idea too. I'll switch it